corehq/apps/reports/dont_use/fields.py

```diff
@@ -260,16 +239,3 @@ class ReportMultiSelectField(ReportSelectField):
 
     def update_params(self):
         self.selected = self.request.GET.getlist(self.slug) or self.default_option
-
-
-class MultiSelectGroupField(GroupFieldMixin, ReportMultiSelectField):
-    default_option = ['_all']
-    placeholder = 'Click to select groups'
-    help_text = "Start typing to select one or more groups"
-
-    @property
-    def options(self):
-        self.groups = Group.get_reporting_groups(self.domain)
-        opts = [dict(val=group.get_id, text=group.name) for group in self.groups]
-        opts.insert(0, {'text': 'All', 'val': '_all'})
-        return opts
```

Comment:
I needed to make some modifications to this filter, and I wasn't about to do it on the old `dont_use.fields` class, so I removed references to these old fields (not used anywhere else)